Libraries/Blob/RCTBlobManager.m

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2015-2016 Silk Labs, Inc.
+ * All Rights Reserved.
+ */
+
+#import "RCTBlobManager.h"
+
+#import <React/RCTConvert.h>
+
+static NSString *const kBlobUriScheme = @"blob";
+
+@implementation RCTBlobManager
+{
+  NSMutableDictionary<NSString *, NSData *> *_blobs;
+  NSOperationQueue *_queue;
+}
+
+RCT_EXPORT_MODULE(BlobModule)
+
+- (NSDictionary<NSString *, id> *)constantsToExport
```

Comment:
That would introduce two places where the string is defined, one on native side (we need to refer to `BLOB_URI_SCHEME` and second on the Javascript side). Also, on Android it has a different value. 

@satya164 ?